src/Table/index.jsx

```diff
@@ -68,14 +71,15 @@ const Table = ( {
         return React.cloneElement( row,
             {
                 children : cells.map( ( cell, index ) =>
-                        {
+                {
                     if ( typeof columns[ index ] === 'object' )
-                            {
+                    {
                         const title = columns[ index ].title;
                         const size  = columns[ index ].size;
 
                         return React.cloneElement( cell,
                             {
+                                className   : cssMap.cell,
```

Comment:
There’s a small problem with this:

In theory, the cell might already have some custom `className`, that we’d like to preserve. For example you could set the `children` prop of the table to the following:
```
<TableRow className="customRowClass">
    <TableCell className="customCellClass1">abc</TableCell>
    <TableCell className="customCellClass2">123</TableCell>
    ...
</TableRow>
...
```

Setting `className` to `cssMap.cell` would override any custom `className`.

Check rows 91-92 below to see how custom `className` can be preserved.